src/Features/VisualBasic/Portable/GenerateConstructor/VisualBasicGenerateConstructorService.vb

```diff
@@ -249,5 +249,29 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.GenerateConstructor
             Dim newArguments = oldArgumentList.Arguments.Take(argumentCount)
             Return SyntaxFactory.ArgumentList(New SeparatedSyntaxList(Of ArgumentSyntax)().AddRange(newArguments))
         End Function
+
+        Protected Overrides Function CanDelegeteThisConstructor(state As State, document As SemanticDocument, delegatedConstructor As IMethodSymbol, Optional cancellationToken As CancellationToken = Nothing) As Boolean
+            Dim currentConstructor = document.SemanticModel.GetDeclaredSymbol(state.Token.GetAncestor(Of ConstructorBlockSyntax)().SubNewStatement, cancellationToken)
```

Comment:
most of this method is equivalent tot eh c# version.  we should share as much as possible, andonly use the subclassing for thepart that deviates (getting the constructor forwarding node).